CMakeLists.txt

```diff
@@ -386,6 +386,8 @@ if(${USE_TETGEN_NONFREE_TPL} MATCHES ON )
   find_package(Tetgen REQUIRED)
 endif(${USE_TETGEN_NONFREE_TPL} MATCHES ON )
 
+set(USE_MPI OFF CACHE BOOL "Build Kratos with MPI")
```

Comment:
this message is a bit misleading no? i understand that's the flag to control it, but the way it is expressed is a bit strange

Reply:
How about ```"Build Kratos with MPI suport?"```